lib/utils/parse-argv.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const path = require('path');
+
+module.exports = function parseArgv(command, argv = {}) {
```

Comment:
argv has already been parsed, right? So maybe there's a name that better describes what this function does?